[9.0][ADD] backport of account_journal_lock_date to 9.0

robinkeunen opened the pull request:
Allow locking each account journals independently. 

This is a backport of version 10.0 module [account_journal_lock_date](https://github.com/OCA/account-financial-tools/tree/10.0/account_journal_lock_date) by @sbidoul  .

1st pull request for the OCA, best practice or style comments welcome :-)

robinkeunen committed:
[ADD] backport of account_journal_lock_date to 9.0

pedrobaeza commented:
Welcome and thanks for contributing. There are several things to do:

* Sign the CLA
* Preserve commit history. Use https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-9.0, but changing the source branch to the proper one.

robinkeunen committed:
[ADD] README.rst

robinkeunen committed:
[REM] data tag from view declaration

robinkeunen commented:
Hi Pedro, thank you for your answer.

I sent the signed CLA a few minutes ago. I also added a readme.rst and removed the data tag as advised in the link you referenced.

> Preserve commit history. Use https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-9.0, but changing the source branch to the proper one.

I am not sure to understand. I used OCA:9.0 as my source branch, should I use another one?

pedrobaeza commented:
I'm talking about importing commit history using the technical method explained there, but on the part where `8.0` is mentioned, you should put your source branch (10.0?)

pedrobaeza commented:
Superseeded by #685
